### ISSUE TITLE
curl package SSL fix

### ISSUE THREAD

**Issue by botmtl** [chocolatey/chocolatey-coreteampackages#957]
curl is somewhat "crippled" when installed as is with cinst curl.  

## Expected Behavior
ssl should be usable with curl

## Current Behavior
error message with any https: curl failed to verify the legitimacy of the server

## Possible Solution
curl.haxx.se now publishes an up-to-date ca bundle to use with the curl utility.
the latest version of said bundle is available at https://curl.haxx.se/ca/cacert.pem
curl.exe looks for said bundle in it's own folder (chocolatey\lib\curl\tools\curl_x64) in our case
There is a twist, it expects the bundle (cacert.pem) to be named curl-ca-bundle.crt

TLDR:
download https://curl.haxx.se/ca/cacert.pem 
move-rename cacert.pem to curl\tools\curl_x64\curl-ca-bundle.crt

[ref to page](https://curl.haxx.se/docs/sslcerts.html) mentioning that curl checks for a file called _curl-ca-bundle.crt_ in it's own folder.
I can personally attest that putting the the ca-cert bundle in the curl.exe restores ssl functionality for curl.

## Your Environment
* Package Version used: curl v7.58.0
* Operating System and version: windows
* Chocolatey version: Chocolatey v0.10.8

**Comment by AdmiringWorm** [comment]
the file needed to fix ssl problems is now being included in the package (should be available on chocolatey.org shortly), let me know if there are any more problems

**Comment by botmtl** [comment]
thanks @AdmiringWorm, works great.

**Comment by vszakats** [comment]
8.10.0 and later curl-for-win builds now embed the CA bundle in
`curl.exe` itself. Meaning it works without an external file.

Ref: https://github.com/curl/curl/pull/14582

**Comment by pauby** [comment]
@vszakats this issue is closed.
